xml/zypper.xml

```diff
@@ -643,16 +651,32 @@ Severity    : important
 Created On  : Mon 16 Aug 2021 03:44:00 AM PDT
 Interactive : ---
 Summary     : Security update for cpio
-Description : 
+Description :
     This update for cpio fixes the following issues:
 
-    It was possible to trigger Remote code execution due to a integer overflow 
+    It was possible to trigger Remote code execution due to a integer overflow
     (CVE-2021-38185, bsc#1189206)
-
-    UPDATE:
-    This update was buggy and could lead to hangs, so it has been retracted. 
-    There will be a follow up update.
-    [...]</screen>   
+Provides    : patch:SUSE-SLE-Product-SLES-15-2021-2689 = 1
+Conflicts   : [99]
+    libsystemd0.x86_64 &lt; 246.16-150300.7.36.1
+    libsystemd0.noarch &lt; 246.16-150300.7.36.1
+    libsystemd0-32bit.x86_64 &lt; 246.16-150300.7.36.1
+    libsystemd0-32bit.noarch &lt; 246.16-150300.7.36.1
+    [...]</screen>
+       <para>
+        The above patch conflicts with the affected/vulnerable versions of some
+        packages. If any of these affected/vulnerable packages are installed, a
+        conflict is triggered and the patch is classified as
+        <emphasis>needed</emphasis>. <command>zypper patch</command> tries to
+        install all available patches and If it encounters problems, it reports
+        them so that you know that not all updates are installed. The conflict
+        can be resolved by either updating the affected/vulnerable packages or
+        by removing them. Because our update repositories also ship fixed
+        packages, updating is the common solution. If the package cannot be
+        updated&mdash;for example, due to dependency issues or package
+        locks&mdash;it needs be deleted. Such issue is reported and needs to be
+        confirmed by the user.
```

Comment:
```suggestion
        locks&mdash;it must be deleted.
```

" Such issue is reported and needs to be confirmed by the user." is confusing. Reported where? Issue needs to be confirmed by the user? Confirmed how? And then what?